dev/preview/previewctl/cmd/stale.go

```diff
@@ -67,7 +67,7 @@ func newListStaleCmd(logger *logrus.Logger) *cobra.Command {
 }
 
 func (o *listWorkspaceOpts) listWorskpaceStatus(ctx context.Context) ([]preview.Status, error) {
-	branches, err := preview.GetRecentBranches(time.Now().AddDate(0, 0, -3))
+	branches, err := preview.GetRecentBranches(time.Now().AddDate(0, 0, -30))
```

Comment:
Nit: There is a log line further down with "branch doesn't exist, or last commit older than 3 days". Could you move this into a constant instead and use it both places? ☺️

Reply:
Thanks for spotting this, I updated the corresponding log line accordingly. I somehow had difficulty naming the variable and would have to negate it in the first occurrence. So it felt too much to put it into a var to me.